terraform/aws/analytical-platform-development/control-panel-message-broker/iam-policies.tf

```diff
@@ -0,0 +1,83 @@
+resource "aws_sqs_queue_policy" "s3_queue_iam_policy" {
```

Comment:
It's way better to use the [policy document datasource](https://registry.terraform.io/providers/hashicorp/aws/2.61.0/docs/data-sources/iam_policy_document) than inlining policies like this: